test/integration/scheduler/scheduler_test.go

```diff
@@ -785,7 +758,7 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			framework.ExpectResourceFlavorToBeDeleted(ctx, k8sClient, onDemandFlavor, true)
 		})
 
-		ginkgo.It("Should schedule workloads by their priority strictly in StrictFIFO", func() {
+		ginkgo.It("Should schedule workloads by their priority strictly", func() {
```

Comment:
No need to say StrictFIFO again, because this is inside `When("Queueing with StrictFIFO")`